exonum/src/lib.rs

```diff
@@ -18,6 +18,26 @@
 
 #![deny(missing_debug_implementations, missing_docs, unsafe_code, bare_trait_objects)]
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
+#![cfg_attr(feature = "cargo-clippy", deny(clippy_pedantic))]
+#![cfg_attr(
+    feature = "cargo-clippy",
+    allow(
+          option_map_unwrap_or_else,
+          // Next cast.. lints don't give alternatives.
+          cast_possible_wrap, cast_possible_truncation, cast_sign_loss,
+          // The lint does not work properly with desugaring and macro.
+          used_underscore_binding,
+          // filter(..).map(..) often looks more shorter and readable.
+          filter_map,
+          // Next lints produce too much noise.
+          stutter, similar_names,
+          // Variant name ends with the enum's name. Similar behavior to similar_names.
+          pub_enum_variant_names,
+          // Next lints allowed due to false possitive.
+          doc_markdown,
+          use_self, // can be enabled when rust-lang-nursery/rust-clippy#2894 is fixed.
```

Comment:
Please either move this line above (if you want to focus that this lint is disabled because of its incompleteness) or add `TODO` (if you want to highlight that this lint is temporary disabled).